### DISCUSSION TITLE
Animated hit300 not matching stable when particle300.png is present in skin folder

### DISCUSSION THREAD

**Discussion by Aksumxd** [ppy/osu#16171] [Q&A] [answered]
The presence of the particle300.png file causes animated hit300s to look different in stable. In lazer this is not the case. 

Hit300 animation in stable release:
![stablehit300](https://user-images.githubusercontent.com/63202855/146740295-3512bc0f-2439-48d0-bbc4-dce5c9b1670e.gif)

Hit300 animation in lazer:
![lazerhit300](https://user-images.githubusercontent.com/63202855/146740408-c9b6cbf5-e93f-467c-98c6-90c91b545916.gif)

Skin used in the example: [Download](https://dl.dropboxusercontent.com/s/88sya2rf0j9f07v/-%20-%20Aksum%20Freezd.osk)
(this is most likely the case for hit100, hit50, & hit0 as well)

Not sure whether this is intended behaviour or not.

**Comment by bdach** [comment, marked as answer]
#16173
